pages/input/index.vue

```diff
@@ -119,6 +120,8 @@
 </template>
 
 <script>
+import _ from 'lodash'
```

Comment:
kalo cuma butuh function `find` nya aja, kamu bisa `import _find from 'lodash/find'` itu paling jadi 2Kb daripada load seluruh lodash yang sekitar 100Kb, more info here : https://stackoverflow.com/questions/35250500/correct-way-to-import-lodash#35251059